content/docs/command-reference/metrics/index.md

```diff
@@ -80,30 +127,24 @@ $ dvc metrics show -a
 		}
 ```
 
-We can also give DVC an `xpath` for the metric file, so that it outputs only the
-AUC value. For JSON metrics, we use
-[JSONPath](https://goessner.net/articles/JsonPath/index.html) expressions to
-filter data out of metric files:
-
-```dvc
-$ dvc metrics show --xpath AUC data/eval.json
-      data/eval.json: {'AUC': 0.65115}
-```
-
-The `xpath` filter can be saved as the default way to display a metrics file:
+When metrics file changes in the user workspace witout commiting it to the Git,
+the `dvc metrics diff` command show the difference between metrics values:
 
 ```dvc
-$ dvc metrics modify data/eval.json --xpath AUC
-$ dvc metrics show
-      data/eval.json: {'AUC': 0.65115}
+$ dvc metrics diff
+Path       Metric    Value    Change
+eval.json  ACU       0.66729  0.01614
+eval.json  error     0.16982  0.00322
+eval.json  TP        516      -12
 ```
 
-And finally let's remove `data/eval.json` from the project metrics:
+Metrics commited to the Git history can be shown by referencing the commit name
+or any aliase:
```

Comment:
```suggestion
Metric files committed with Git can be shown by referencing the commit,
branch, or tag (or any other reference):
```
Link reference to the corresponding Git doc (revision I think? Need to check the other docs where we link to git-scm.com).

Reply:
Or maybe skip the last part about refs.